schema/test/unit/computed_columns/application_revision_ciip_incentive_test.sql

```diff
@@ -440,8 +440,8 @@ select throws_like(
       (select * from record)
     )
   $$,
-  '%Reported product requires exported_heat%',
-  'throws an exception if a product requires exported heat and it is not present'
+  '%Reported product requires sold_heat%',
```

Comment:
You didn't update the error message thrown in `schema/deploy/database_functions/validate_energy_products.sql`